packages/web/src/scss/tools/_links.scss

```diff
@@ -1,9 +1,38 @@
+@use '@tokens' as tokens;
+@use '../settings/links';
+
+@mixin base($set-color: false) {
+    text-decoration-skip-ink: links.$text-decoration-skip-ink;
+    text-underline-offset: links.$text-underline-offset;
+
+    @if $set-color {
+        color: tokens.$action-link-primary-default;
+    }
+
+    @media (hover: hover) {
+        &:hover {
+            text-decoration: links.$text-decoration-hover;
+
+            @if $set-color {
+                color: tokens.$action-link-primary-hover;
+            }
+        }
+    }
+
+    &:active {
+        text-decoration: links.$text-decoration-hover;
+
+        @if $set-color {
+            color: tokens.$action-link-primary-active;
+        }
+    }
+}
+
 @mixin stretch($pseudo-element: before) {
     &::#{$pseudo-element} {
         content: '';
         position: absolute;
         z-index: 0;
-        display: block;
```

Comment:
Are you sure this won't be potential BC?

Reply:
No worry, it won't be. I cannot find anything at MDN, so this is what SO says:

> All elements that are position:absolute; are automatically treated as display:block, since that's the only logical display mode for absolute positioning.

https://stackoverflow.com/a/5042666